pkg/assembler/backends/neo4j/src.go

```diff
@@ -549,3 +525,34 @@ func setSrcMatchValues(sb *strings.Builder, src *model.SourceSpec, objectSrc boo
 		}
 	}
 }
+
+func generateModelSource(srcType, namespaceStr, nameStr string, commitValue, tagValue interface{}) model.Source {
```

Comment:
Should we return pointers instead of copying the struct? Some of these are quite large, so we might get slowdown if we keep copying them on functions calls and returns.

Reply:
Done.